### ISSUE TITLE
Issue with embedding a picture.

### ISSUE THREAD

**Issue by manzoniUC** [cfss-su20/hw01#12]
Hello,
Mint helped me understand why my picture wasn't rendering in the RStudio preview window, but it's still not showing up on Github (as in, the picture isn't there when I go to Github and look at what is supposed to be the entirely formatted bio).
Thank you!

**Comment by bensoltoff** [comment]
Did you stage and commit the image in your repo? I don't see it in your fork on GitHub.

**Comment by manzoniUC** [comment]
I was under the impression I had done that, but I guess not. I've gone ahead and staged and committed the image, and it's still not appearing on GitHub. Thanks!

**Comment by bensoltoff** [comment]
Nope, it's not up there. You should see a checkbox in the Git panel within RStudio next to the name of your image file. Check that box to stage the file, then you need to commit it with a brief message. Finally, push that commit up to GitHub.

**Comment by bensoltoff** [comment]
That said, you are using the correct Markdown formatting to include the image in your README. As long as you successfully commit and push the image to GitHub, it should work

**Comment by manzoniUC** [comment]
I was misunderstanding your instruction. I just now understood that you meant for me to commit the image itself rather than the README file containing said image. Sorry about that, and thank you!

**Comment by bensoltoff** [comment]
No problem! If the image is not already hosted somewhere on the internet, this is a good technique for publishing it and making it accessible in the repository

**Comment by bensoltoff** [comment]
I'll leave the issue open for now in case other students have a similar problem incorporating images into their README

**Comment by shuoq25** [comment]
Is it possible to generate an URL for the image (e.g. upload the image to imgbb.com and copy the image link) and embed the URL instead of the image?

**Comment by bensoltoff** [comment]
Yes you could do that instead

**Comment by bensoltoff** [comment]
The important thing is that it is hosted somewhere on a website and not just locally on your computer. No one else would be able to access it if it is just on your computer
